tasks/eth/009-fp-upgrade/README.md

```diff
@@ -1,6 +1,6 @@
 # Mainnet Fault Proofs Upgrade
 
-STATUS: READY TO SIGN
+Status: [EXECUTED](https://etherscan.io/tx/0xc721928a1b418053eb651b56380b266bcb4df92ba26201efddb2db4c0e36f3e7)
```

Comment:
This looks like it's just the approveHash call, we should get the actual execution call here

Reply:
```suggestion
Status: [EXECUTED](https://etherscan.io/tx/0xbe810ed6e1e827930bd46cde42c4883662ddb6d244a26af2f0409cc2782bee61)
```